homeassistant/components/heos/strings.json

```
@@ -2,13 +2,23 @@
   "config": {
     "step": {
       "user": {
-        "title": "Connect to Heos",
-        "description": "Please enter the host name or IP address of a Heos device (preferably one connected via wire to the network).",
+        "title": "Connect to HEOS",
+        "description": "Please enter the host name or IP address of a HEOS-capable product to access your HEOS System.",
         "data": {
           "host": "[%key:common::config_flow::data::host%]"
         },
         "data_description": {
-          "host": "The hostname or IP address of your HEOS device."
+          "host": "Host name or IP address of a HEOS-capable product (preferrably one connected via wire to the network)."
+        }
+      },
+      "reconfigure": {
+        "title": "Reconfigure HEOS",
+        "description": "Complete this form to change the host name or IP address of the HEOS-capable product used to access your HEOS System.",
```

```suggestion
        "description": "Change the host name or IP address of the HEOS-capable product used to access your HEOS System.",
```

just a bit more concise imo